### ISSUE TITLE
pyldavis.display() not working in Jupyterlab 0.32.0

### ISSUE THREAD

**Issue by cwrather** [bmabey/pyLDAvis#121]
pyldavis.display() works fine in Jupyter Notebook 5.0.0. In Jupyterlab 0.32.0 it does not give an error, but it also does NOT display the graphic. pyLDAvis version 2.1.1.

I also left a bug report with the Jupyterlab team.

**Comment by cwrather** [comment]
jasongrout the maintainer of Jupyterlab said the following:

I looked into the pyldavis codebase a bit. It looks like it is using custom Javascript that is very specific to the classic Notebook environment, and it will need to be ported over to work in JupyterLab (probably as a JupyterLab extension). Can you open an issue over in the pyldavis repo about porting to JupyterLab?

**Comment by bmabey** [comment]
Yeah, when `pyldavis` was written I don't think JupyterLab was even in beta. I haven't looked at what would be needed to support JupyterLab but I imagine the changes wouldn't be huge.

**Comment by cwrather** [comment]
I think the Jupyter team is planning to deprecate Jupyter notebook in favor of JupyterLab at some point down the line.

**Comment by dbl001** [comment]
pyldavis does not display in Jupyter lab 1.2.6 with the on server process running on Ubuntu 16.06lts  and Jupyter lab on OS X 10.11.6 running Anaconda python 3.6 .
The output is blank.
![Screen Shot 2020-02-16 at 8 02 13 AM](https://user-images.githubusercontent.com/3105499/74608110-394d4000-5093-11ea-9e80-5d6f23bd67ff.png)
